framework/src/outputs/ConsoleUtils.C

```diff
@@ -99,51 +99,55 @@ outputMeshInformation(FEProblemBase & problem, bool verbose)
   std::stringstream oss;
   oss << std::left;
 
-  MooseMesh & moose_mesh = problem.mesh();
-  MeshBase & mesh = moose_mesh.getMesh();
+  const MooseMesh & mesh = problem.mesh();
+
+  const auto fe_backend = problem.feBackend();
 
   if (verbose)
   {
-    bool forced = moose_mesh.isParallelTypeForced();
-    bool pre_split = moose_mesh.isSplit();
-
-    // clang-format off
-    oss << "\nMesh: " << '\n'
-        << std::setw(console_field_width)
-        << "  Parallel Type: " << (moose_mesh.isDistributedMesh() ? "distributed" : "replicated")
-        << (forced || pre_split ? " (" : "")
-        << (forced ? "forced" : "")
-        << (forced && pre_split ? ", " : "")
-        << (pre_split ? "pre-split" : "")
-        << (forced || pre_split ? ")" : "")
-        << '\n'
-        << std::setw(console_field_width) << "  Mesh Dimension: " << mesh.mesh_dimension() << '\n'
-        << std::setw(console_field_width) << "  Spatial Dimension: " << mesh.spatial_dimension()
+    bool forced = mesh.isParallelTypeForced();
+    bool pre_split = mesh.isSplit();
+
+    oss << "\nMesh: " << '\n' << std::setw(console_field_width);
+
+    if (fe_backend == Moose::FEBackend::LibMesh)
+    {
+      oss << "  Parallel Type: " << (mesh.isDistributedMesh() ? "distributed" : "replicated")
```

Comment:
is the mesh always distributed in mfem?

Reply:
yes

Reply:
I suppose for the general MOOSE user new to MFEM that could be helpful